### DISCUSSION TITLE
MUST HAVE HOTKEY FUNCTIONS

### DISCUSSION THREAD

**Discussion by Proton7123** [pear-devs/pear-desktop#949] [Ideas]
1. The ability to rate a song using hotkey, while using a different app.
2. The ability to seek back/forth in currently playing song by holding down a hotkey, again, from another app.
I only listen to music while I play video games, and I am constantly needing to interact with the app without switching windows and using the touchpad. It's totally unnecessary, and I would (ignorantly) assume these changes would be relatively easy to implement.. but I'm just a former agitated lightning sniffer with no money so it's whatever.

**Comment by Araxeus** [comment]
Already possible using the `global` field in the `shortcuts` plugin. See 

https://github.com/th-ch/youtube-music/issues/106

https://github.com/th-ch/youtube-music/blob/v1.19.0/providers/song-controls.js#L12-L51

(Btw both those links are from the readme)
